src/test/java/fi/aalto/cs/apluscourses/integration/ApiTest.java

```diff
@@ -14,19 +17,49 @@
   @ClassRule
   public static final EnvironmentChecker checker = new EnvironmentChecker("CI");
 
+  private static final String BASE_URL = "http://localhost:8000/api/v2/";
+
+  @Test
+  public void getSubmissionsResultsReturns() throws MalformedURLException {
+    final String firstExercise = "modules[0].exercises[0]";
+    final String url = BASE_URL + "courses/100/points/me/";
+
+    given()
+        .auth()
+        .preemptive()
+        .basic("zoralst1", "zoralst1")
+        .when()
+        .contentType(ContentType.JSON)
+        .get(url)
+        .then()
+        .assertThat()
+        .statusCode(HttpStatus.SC_OK)
+        .body("modules[0].id", equalTo(10))
```

Comment:
FFS, so this is completely different from the `local` id (keep in mind and also you might need [this](https://docs.travis-ci.com/user/running-build-in-debug-mode/#restarting-a-job-in-debug-mode-via-api))